src/storage/admin/RebuildEdgeIndexProcessor.cpp

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "storage/admin/RebuildEdgeIndexProcessor.h"
+
+namespace nebula {
+namespace storage {
+
+void RebuildEdgeIndexProcessor::process(const cpp2::RebuildIndexRequest& req) {
+    CHECK_NOTNULL(kvstore_);
+    auto space = req.get_space_id();
+    auto parts = req.get_parts();
+    auto edgeType = req.get_schema_id().get_edge_type();
+    auto edgeVersion = req.get_schema_version();
+    auto indexID = req.get_index_id();
+    LOG(INFO) << "Rebuild Edge Index Space " << space << " Edge Type " << edgeType
+              << " Edge Index " << indexID << " Edge Verison " << edgeVersion;
+
+    for (PartitionID part : parts) {
+        // firstly remove the discard index
+        auto indexPrefix = NebulaKeyUtils::indexPrefix(space, indexID);
+        doRemovePrefix(space, part, std::move(indexPrefix));
+
+        std::unique_ptr<kvstore::KVIterator> iter;
+        auto prefix = NebulaKeyUtils::prefix(part);
```

Comment:
This result set is too big.  I have a bold idea to split the NebulaKeyType from kData to kVertexData and kEdgeData. WDYT?

enum class NebulaKeyType : uint32_t {
    kData              = 0x00000001,
    kIndex             = 0x00000002,
    kUUID              = 0x00000003,
    kSystem            = 0x00000004,
};